packages/integrations/src/jellyfin/jellyfin-integration.ts

```diff
@@ -0,0 +1,57 @@
+import { Jellyfin } from "@jellyfin/sdk";
+import { getSessionApi } from "@jellyfin/sdk/lib/utils/api/session-api";
+
+import { Integration } from "../base/integration";
+import type { StreamSession } from "../interfaces/media-server/session";
+
+const jellyfin = new Jellyfin({
```

Comment:
Are you sure that you want to initalize a new instance of jellyfin here? This code will run always when anything from homarr/integrations is imported. If this connects to Jellyfin it should be moved inside the JellyfinIntegration class